docs/deploy.md

```diff
@@ -97,35 +108,38 @@ SEE ABOVE FOR WARNINGS ABOUT USING GIT FOR THIS.
 
 # Deploying a prepared cluster
 
-## Cloning existing cluster configuration
-
-Skip this step if you're starting a new cluster.
+## Cloning an existing cluster configuration
 
 To download existing configuration:
 
     $ export HOMEWORLD_DIR="$HOME/my-cluster"
     $ export HOMEWORLD_DISASTER="/media/usb-crypt/homeworld-disaster"
-    $ git clone git@github.com:sipb/hyades-cluster $HOMEWORLD_DIR
+    $ git clone git@github.mit.edu:sipb/hyades-cluster $HOMEWORLD_DIR
 
 Make sure to verify that you have the correct commit hash, out of band.
 
-## Set SSH configuration
+## Extracting upstream keys
+
+For the development cluster, we already have upstream keys. Extract them from the configuration like so:
+
+    $ spire keytab export egg-sandwich egg-keytab
+    $ spire https export homeworld.mit.edu ./homeworld.mit.edu.key ./homeworld.mit.edu.pem
+
```

Comment:
This should go before importing the keys into the new cluster, and only be recommended when rotating the disaster recovery key. There's no point to exporting them if you're not going to use them.